README.md

```diff
@@ -37,6 +39,10 @@ To stop the container by name (if you used the `--name` tag when launching it),
 docker stop groundhog_server
 ```
 
+***
+## Clients
+With the docker container running, choose a client to get and append elevation data to your dataset. 
```

Comment:
Nothing about the clients relies on the fact that the app is running _in .a container_. It could just as easily be running on some dedicated server.

Can you change `With the docker container running,` to `Given a running instance of the service,`?